modules/core/src/main/scala/org/scalasteward/core/application/Cli.scala

```diff
@@ -238,6 +238,15 @@ object Cli {
   private val gitHubApp: Opts[Option[GitHubApp]] =
     (githubAppId, githubAppKeyFile).mapN(GitHubApp.apply).orNone
 
+  private val azureReposOrganization: Opts[Option[String]] =
+    option[String](
+      "azure-repos-organization",
+      "The Azure organization (required when vcs type is azure-repos)"
+    ).orNone
```

Comment:
I think it is better to validate `azure-repos-organization` so that user can understand what is missing.
Could you try to implement validation?
https://ben.kirw.in/decline/usage.html#transforming-and-validating

Reply:
@exoego Thanks for taking a look. I'll implement the validation.

Reply:
@robinraju tried your changes on azure and they work. thanks for it.

I have this proposal for the validation
```
private val azureReposOrganization: Opts[Option[String]] =
    option[String](
      "azure-repos-organization",
      "The Azure organization (required when vcs type is azure-repos)"
    ).validate("required when vcs type is azure-repos")(organization =>
      if (vcsTypeArgument.toString() === "azure-repos") organization.nonEmpty else true
    ).orNone
```

@exoego is this the validation that is missing to complete this PR?

Reply:
@sapka12 👍 
You can create a new PR to complete this PR if you need this feature.

Reply:
https://github.com/scala-steward-org/scala-steward/pull/2837

Reply:
@exoego 
PR: https://github.com/scala-steward-org/scala-steward/pull/2837